src/app/dashboard/commodities/page.tsx

```diff
@@ -61,18 +56,7 @@ export default function CommoditiesPage(): JSX.Element {
         </span>
       </div>
       <div className="grid grid-cols-12">
-        {currencies.map(commodity => (
-          <Link
-            key={commodity.guid}
-            href={`/dashboard/commodities/${commodity.guid}`}
-            className="card col-span-1 cursor-pointer hover:shadow-lg text-slate-400 dark:hover:text-white"
-          >
-            <div className="flex items-center">
-              <BiMoney className="mr-1" />
-              {commodity.mnemonic}
-            </div>
-          </Link>
-        ))}
+        {currencies.map(commodity => <CommodityCard key={commodity.guid} guid={commodity.guid} />)}
```

Comment:
new CommodityCard component that contains all logic about the commodity card (including the warning, tooltip, etc)